File: SalesTaxCalculationKata.Core/Models/OrderModel.cs
using System.Collections.Generic;

namespace SalesTaxCalculationKata.Core.Models
{
    public class OrderModel
    {
        public int OrderId { get; set; }
        public List<OrderItemModel> OrderItems { get; set; } = new List<OrderItemModel>();
        public decimal SalesTaxTotal { get; set; }
        public decimal GrandTotal { get; set; }
        public bool IsComplete { get; set; }
    }
}

File: SalesTaxCalculationKata.Web/src/models/order-model.ts
export class OrderModel {
  orderId: number;
  orderItems: Array<OrderItemModel>;
  salesTaxTotal: number;
  grandTotal: number;
  isComplete: boolean;
}

export class OrderItemModel {
  orderItemId: number;
  productId: number;
  productPrice: number;
  productDescription: string;
  salesTax: number;
}
